index.js

```diff
@@ -6,6 +6,9 @@ var styleCheck = /\srel=["\'][^"]*stylesheet[^"]*["\']/;
 var srcCheck = /\ssrc=["\']([^"\']+)["\']/;
 var hrefCheck = /\shref=["\']([^"\']+)["\']/;
 
+// node 0.10
+require('es6-promise').polyfill();
```

Comment:
we need to be careful not to force `Promise` to be present, that isn't going to make users happy.


Any lib that uses Promise, should just include some lib (like RSVP and use it directly until we drop drop 10